.gitignore

```diff
@@ -24,4 +24,8 @@ yarn-debug.log*
 yarn-error.log*
 
 # credentials
+gcloud_creds.json
```

Comment:
Added this in advance to avoid future accidents, since @JonathanChavezTamales and @SofiaVega are using this file to handle credentials.

I noticed some branches already have this, but as early as we get this to master branch, the better.

Reply:
lgtm